SSDynamicText/SSDynamicButton.h

```diff
@@ -4,21 +4,20 @@
 //
 //  Created by Adam Grzegorowski on 18/07/15.
 //
-//
```

Comment:
Ahoy @Grubas7, can you restore this line and its twin in `SSDynamicButton.m`? The build failed because it performs a strict check on file header styles and it is expecting the default 7 lines of comments.